src/devices/thermostat.cpp

```diff
@@ -2423,13 +2496,18 @@ bool Thermostat::set_nofrostmode(const char * value, const int8_t id) {
     if (hc == nullptr) {
         return false;
     }
-
     uint8_t set = 0xFF;
-    if (!Helpers::value2enum(value, set, FL_(enum_nofrostmode))) {
-        return false;
+    if (model() == EMS_DEVICE_FLAG_RC300) {
+        if (Helpers::value2enum(value, set, FL_(enum_controlmode1))) {
```

Comment:
I think this should be ' enum_nofrostmode1' as in register_value

Reply:
Thanks for quality control ... corrected